packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -276,7 +277,16 @@ export function createContainer(
   isConcurrent: boolean,
   hydrate: boolean,
 ): OpaqueRoot {
-  return createFiberRoot(containerInfo, isConcurrent, hydrate);
+  const fiberRoot = createFiberRoot(containerInfo, isConcurrent, hydrate);
+  if (__DEV__) {
+    // jest isn't a 'global', it's just exposed to tests via a wrapped function
+    // further, this isn't a test file, so flow doesn't recognize the symbol. So...
+    // $FlowExpectedError - because requirements don't give a damn about your type sigs.
+    if ('undefined' !== typeof jest) {
```

Comment:
Off-topic: Is there a task somewhere to support other test frameworks besides Jest? What's the plan?

Reply:
we only discussed it offhand. filed https://github.com/facebook/react/issues/15439